src/elements/tip-well/TipWell.js

```diff
@@ -0,0 +1,66 @@
+// NG2
+import { Component } from '@angular/core';
+// App
+import { NOVO_BUTTON_ELEMENTS } from '../button';
+
+@Component({
+    selector: 'novo-tip-well',
+    inputs: [
+        'name',
+        'tip',
+        'buttonText'
+    ],
+    directives: [
+        NOVO_BUTTON_ELEMENTS
+    ],
+    template: `
+        <div *ngIf="isActive">
+            <p>{{ tip }}</p>
+            <button theme="dialogue" (click)="hideTip()">{{ buttonText }}</button>
+        </div>
+    `
+})
+export class TipWell {
+    constructor() {
+        this.isActive = true;
+        // Check if localStorage is enabled
+        this.isLocalStorageEnabled = (() => {
+            let isEnabled = false;
+            if (typeof localStorage === 'object') {
+                try {
+                    localStorage.setItem('lsTest', 1);
+                    localStorage.removeItem('lsTest');
```

Comment:
just a thought @krsween, all tip wells may not need buttons and all tip wells' buttons may not be doing the same things so it will be great if in the future we can make the button optional and add a call back option for what the button would do so that the button action can be controlled by the user of the tip well.